cmd/argoexec/commands/artifacts.go

```diff
@@ -15,7 +15,8 @@ func NewArtifactsCommand() *cobra.Command {
 		Run: func(cmd *cobra.Command, args []string) {
 
 			var artifacts v1alpha1.WithArtifacts
-			err := json.Unmarshal([]byte(args[1]), &artifacts)
+			log.Info("ARGS:", args)
```

Comment:
tip: I add `println("ALEX", args)` for temporary debugging statements as they jump out of me from the logs + easy to remember to remove